Metadata tooltip popup on colum header casues data table to reset for number columns that are all NULLs

**Describe the bug**
New metadata tooltip popups in v0.42.0-rc1 can cause sections of the app to reset if the column is a number type and it only contains NULL values.

The section that refreshes/resets depends where the popup is triggered i.e. table headers, filter selector, dimension picker, and whether the simple or notebook editor is being used.

**Logs**
JS logs:
```
TypeError: Cannot read properties of null (reading 'toFixed')
    at O (FieldFingerprintInfo.jsx:74:67)
    at Qo (react-dom.production.min.js:153:146)
    at xs (react-dom.production.min.js:261:496)
    at yl (react-dom.production.min.js:246:265)
    at ml (react-dom.production.min.js:246:194)
    at ll (react-dom.production.min.js:239:172)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:19:467)
    at Vi (react-dom.production.min.js:122:325)
    at Zi (react-dom.production.min.js:123:61)
overrideMethod @ VM17260 react_devtools_backend.js:4061
os @ react-dom.production.min.js:209
Ms.o.componentDidCatch.n.callback @ react-dom.production.min.js:227
vo @ react-dom.production.min.js:131
us @ react-dom.production.min.js:212
Ml @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
wl @ react-dom.production.min.js:248
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
VM17260 react_devtools_backend.js:4061 Error caught in <Visualization> TypeError: Cannot read properties of null (reading 'toFixed')
    at O (FieldFingerprintInfo.jsx:74:67)
    at Qo (react-dom.production.min.js:153:146)
    at xs (react-dom.production.min.js:261:496)
    at yl (react-dom.production.min.js:246:265)
    at ml (react-dom.production.min.js:246:194)
    at ll (react-dom.production.min.js:239:172)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:19:467)
    at Vi (react-dom.production.min.js:122:325)
    at Zi (react-dom.production.min.js:123:61) {componentStack: '\n    in O\n    in g\n    in DimensionInfostyled__Fie…\n    in t\n    in DragDropContextProvider\n    in a'}
overrideMethod @ VM17260 react_devtools_backend.js:4061
value @ Visualization.jsx:101
Ms.o.componentDidCatch.n.callback @ react-dom.production.min.js:227
vo @ react-dom.production.min.js:131
us @ react-dom.production.min.js:212
Ml @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
wl @ react-dom.production.min.js:248
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
VM17260 react_devtools_backend.js:4061 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=Error%3A%20An%20error%20occurred%20displaying%20this%20visualization.&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at To (react-dom.production.min.js:139:47)
    at react-dom.production.min.js:149:278
    at za (react-dom.production.min.js:173:103)
    at xs (react-dom.production.min.js:265:156)
    at yl (react-dom.production.min.js:246:265)
    at ml (react-dom.production.min.js:246:194)
    at ll (react-dom.production.min.js:239:172)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:19:467)
    at Vi (react-dom.production.min.js:122:325)
overrideMethod @ VM17260 react_devtools_backend.js:4061
os @ react-dom.production.min.js:209
Ms.o.componentDidCatch.n.callback @ react-dom.production.min.js:227
vo @ react-dom.production.min.js:131
us @ react-dom.production.min.js:212
Ml @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
wl @ react-dom.production.min.js:248
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
2VM17260 react_devtools_backend.js:4061 Warning: can't clean query without metadata!
overrideMethod @ VM17260 react_devtools_backend.js:4061
value @ StructuredQuery.ts:339
value @ Question.ts:1197
value @ Question.ts:926
C @ urls.js:95
value @ Navbar.jsx:171
value @ Navbar.jsx:286
Wa @ react-dom.production.min.js:182
Ba @ react-dom.production.min.js:181
xs @ react-dom.production.min.js:263
yl @ react-dom.production.min.js:246
ml @ react-dom.production.min.js:246
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
VM17260 react_devtools_backend.js:4061 Warning: can't clean query without metadata!
overrideMethod @ VM17260 react_devtools_backend.js:4061
value @ StructuredQuery.ts:339
value @ Question.ts:1197
value @ Question.ts:926
C @ urls.js:95
value @ Navbar.jsx:171
value @ Navbar.jsx:286
Wa @ react-dom.production.min.js:182
Ba @ react-dom.production.min.js:181
xs @ react-dom.production.min.js:263
yl @ react-dom.production.min.js:246
ml @ react-dom.production.min.js:246
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
enqueueSetState @ react-dom.production.min.js:132
_.setState @ react.production.min.js:12
l.onStateChange @ connectAdvanced.js:207
notify @ Subscription.js:23
t.notifyNestedSubs @ Subscription.js:62
l.onStateChange @ connectAdvanced.js:204
notify @ Subscription.js:23
t.notifyNestedSubs @ Subscription.js:62
l.onStateChange @ connectAdvanced.js:204
d @ createStore.js:172
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:25
dispatch @ applyMiddleware.js:35
action @ store.js:20
(anonymous) @ actions.js:613
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
ne @ app-main.bundle.js?863fbcce303099f75185:2
a @ app-main.bundle.js?863fbcce303099f75185:2
Promise.then (async)
ne @ app-main.bundle.js?863fbcce303099f75185:2
a @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ actions.js:356
(anonymous) @ store.js:23
(anonymous) @ bindActionCreators.js:3
value @ QueryBuilder.jsx:170
ko @ react-dom.production.min.js:137
Ba @ react-dom.production.min.js:176
xs @ react-dom.production.min.js:263
yl @ react-dom.production.min.js:246
ml @ react-dom.production.min.js:246
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
Show 10 more frames
3react-dom.production.min.js:49 Uncaught Error: Minified React error #188; visit https://reactjs.org/docs/error-decoder.html?invariant=188 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at react-dom.production.min.js:49:67
    at it (react-dom.production.min.js:50:217)
    at Object.t.findDOMNode (react-dom.production.min.js:289:336)
    at s.value (ExplicitSize.jsx:31:23)
    at ExplicitSize.jsx:101:28
    at c (throttle.js:32:21)
(anonymous) @ react-dom.production.min.js:49
it @ react-dom.production.min.js:50
t.findDOMNode @ react-dom.production.min.js:289
value @ ExplicitSize.jsx:31
(anonymous) @ ExplicitSize.jsx:101
c @ throttle.js:32
setTimeout (async)
value @ ExplicitSize.jsx:44
us @ react-dom.production.min.js:212
Ml @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
wl @ react-dom.production.min.js:248
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
VM17260 react_devtools_backend.js:4061 Warning: can't clean query without metadata!
overrideMethod @ VM17260 react_devtools_backend.js:4061
value @ StructuredQuery.ts:339
value @ Question.ts:1197
value @ Question.ts:926
C @ urls.js:95
value @ Navbar.jsx:171
value @ Navbar.jsx:286
Wa @ react-dom.production.min.js:182
Ba @ react-dom.production.min.js:181
xs @ react-dom.production.min.js:263
yl @ react-dom.production.min.js:246
ml @ react-dom.production.min.js:246
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
enqueueSetState @ react-dom.production.min.js:132
_.setState @ react.production.min.js:12
l.onStateChange @ connectAdvanced.js:207
notify @ Subscription.js:23
t.notifyNestedSubs @ Subscription.js:62
l.onStateChange @ connectAdvanced.js:204
notify @ Subscription.js:23
t.notifyNestedSubs @ Subscription.js:62
l.onStateChange @ connectAdvanced.js:204
d @ createStore.js:172
(anonymous) @ middleware.js:22
(anonymous) @ index.js:28
(anonymous) @ store.js:25
dispatch @ applyMiddleware.js:35
action @ store.js:20
(anonymous) @ actions.js:1287
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
ne @ app-main.bundle.js?863fbcce303099f75185:2
a @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ actions.js:1220
(anonymous) @ store.js:23
dispatch @ applyMiddleware.js:35
(anonymous) @ actions.js:628
setTimeout (async)
(anonymous) @ actions.js:625
u @ runtime.js:63
(anonymous) @ runtime.js:293
(anonymous) @ runtime.js:118
ne @ app-main.bundle.js?863fbcce303099f75185:2
a @ app-main.bundle.js?863fbcce303099f75185:2
Promise.then (async)
ne @ app-main.bundle.js?863fbcce303099f75185:2
a @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ app-main.bundle.js?863fbcce303099f75185:2
(anonymous) @ actions.js:356
(anonymous) @ store.js:23
(anonymous) @ bindActionCreators.js:3
value @ QueryBuilder.jsx:170
ko @ react-dom.production.min.js:137
Ba @ react-dom.production.min.js:176
xs @ react-dom.production.min.js:263
yl @ react-dom.production.min.js:246
ml @ react-dom.production.min.js:246
ll @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Vi @ react-dom.production.min.js:122
Zi @ react-dom.production.min.js:123
Ki @ react-dom.production.min.js:122
rl @ react-dom.production.min.js:230
_a @ react-dom.production.min.js:163
onMount @ TippyPopover.tsx:43
(anonymous) @ tippy.esm.js:756
D @ tippy.esm.js:754
s @ tippy.esm.js:1434
(anonymous) @ createPopper.js:217
Promise.then (async)
(anonymous) @ createPopper.js:215
Z @ tippy.esm.js:1161
(anonymous) @ tippy.esm.js:1197
show @ tippy.esm.js:1444
(anonymous) @ tippy.esm.js:1230
setTimeout (async)
ee @ tippy.esm.js:1229
U @ tippy.esm.js:1008
Show 16 more frames
VM17260 react_devtools_backend.js:4061 Warning: can't clean query without metadata!
```




**To Reproduce**
Steps to reproduce the behavior:
1. Create a table with and int column that is all NULL values. Mine looks like this in SQL Server
![image](https://user-images.githubusercontent.com/4504437/151288608-da1c0133-4ae6-42a7-9bd4-a3e074850524.png)
2. Sync DB schema
3. New question for that table, visualised as a table
3. Hover the column header for your column of all NULL ints
4. Data table will refresh

**Expected behavior**
A tooltip should be presented as with other columns.

**Screenshots**
![metabase_issue](https://user-images.githubusercontent.com/4504437/151289560-dbedce51-90a5-493e-9853-208dc3c27a72.gif)

**Information about your Metabase Installation:**
```
{
  "browser-info": {
    "language": "en-NZ",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.99 Safari/537.36",
    "vendor": "Google Inc."
  },
  "system-info": {
    "file.encoding": "Cp1252",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "13.0.2+8",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "https://java.oracle.com/",
    "java.version": "13.0.2",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "13.0.2+8",
    "os.name": "Windows 10",
    "os.version": "10.0",
    "user.language": "en",
    "user.timezone": "Pacific/Auckland"
  },
  "metabase-info": {
    "databases": [
      "h2",
      "clickhouse",
      "sqlserver"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "12.1"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.23"
      }
    },
    "run-mode": "prod",
    "version": {
      "date": "2022-01-20",
      "tag": "v0.42.0-rc1",
      "branch": "release-x.42.x",
      "hash": "2af85ba"
    },
    "settings": {
      "report-timezone": "UTC"
    }
  }
}
```

**Severity**
Low, the number of columns that meet this criteria in our system are low but greater than 0. It does make it hard to work with tables that contain them though as you can very easily accidentally trigger a refresh.

This doesn't seem to trigger for varchar columns.

--- a/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
+++ b/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
@@ -71,9 +71,18 @@ function NumberFingerprint({ className, field }) {
   }
 
   const { avg, min, max } = numberFingerprint;
-  const fixedAvg = formatNumber(Number.isInteger(avg) ? avg : avg.toFixed(2));
-  const fixedMin = formatNumber(Number.isInteger(min) ? min : min.toFixed(2));
-  const fixedMax = formatNumber(Number.isInteger(max) ? max : max.toFixed(2));
+
+  function roundNumber(num, digits = 2) {
+    if (num === null) {
+      return "";
+    }
+
+    return formatNumber(Number.isInteger(num) ? num : num.toFixed(digits));
+  }
+
+  const fixedAvg = roundNumber(avg);
+  const fixedMin = roundNumber(min);
+  const fixedMax = roundNumber(max);
 
   return (
     <Table className={className}>

--- a/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
+++ b/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
@@ -67,11 +67,11 @@ function DateTimeFingerprint({ className, field }) {
 
 /**
  * @param {(number|null|undefined)} num - a number value from the type/Number fingerprint; might not be a number
- * @returns {[boolean, string]} - a tuple, [isNumber, formattedNumber]
+ * @returns {[boolean, string]} - a tuple, [isFormattedNumber, formattedNumber]
  */
 function roundNumber(num) {
   if (num == null) {
-    return [false, null];
+    return [false, ""];
   }
 
   return [true, formatNumber(Number.isInteger(num) ? num : num.toFixed(2))];

--- a/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
+++ b/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
@@ -1,6 +1,7 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { t } from "ttag";
+import _ from "underscore";
 
 import { formatDateTimeWithUnit, formatNumber } from "metabase/lib/formatting";
 import Field from "metabase-lib/lib/metadata/Field";
@@ -64,6 +65,18 @@ function DateTimeFingerprint({ className, field }) {
   );
 }
 
+/**
+ * @param {(number|null|undefined)} num - a number value from the type/Number fingerprint; might not be a number
+ * @returns {[boolean, string]} - a tuple, [isNumber, formattedNumber]
+ */
+function roundNumber(num) {
+  if (num == null) {
+    return [false, null];
+  }
+
+  return [true, formatNumber(Number.isInteger(num) ? num : num.toFixed(2))];
+}
+
 function NumberFingerprint({ className, field }) {
   const numberFingerprint = field.fingerprint.type?.["type/Number"];
   if (!numberFingerprint) {
@@ -71,37 +84,30 @@ function NumberFingerprint({ className, field }) {
   }
 
   const { avg, min, max } = numberFingerprint;
+  const [isAvgNumber, formattedAvg] = roundNumber(avg);
+  const [isMinNumber, formattedMin] = roundNumber(min);
+  const [isMaxNumber, formattedMax] = roundNumber(max);
 
-  function roundNumber(num, digits = 2) {
-    if (num === null) {
-      return "";
-    }
+  const someNumberIsDefined = isAvgNumber || isMinNumber || isMaxNumber;
 
-    return formatNumber(Number.isInteger(num) ? num : num.toFixed(digits));
-  }
-
-  const fixedAvg = roundNumber(avg);
-  const fixedMin = roundNumber(min);
-  const fixedMax = roundNumber(max);
-
-  return (
+  return someNumberIsDefined ? (
     <Table className={className}>
       <thead>
         <tr>
-          <th>{t`Average`}</th>
-          <th>{t`Min`}</th>
-          <th>{t`Max`}</th>
+          {isAvgNumber && <th>{t`Average`}</th>}
+          {isMinNumber && <th>{t`Min`}</th>}
+          {isMaxNumber && <th>{t`Max`}</th>}
         </tr>
       </thead>
       <tbody>
         <tr>
-          <td>{fixedAvg}</td>
-          <td>{fixedMin}</td>
-          <td>{fixedMax}</td>
+          {isAvgNumber && <td>{formattedAvg}</td>}
+          {isMinNumber && <td>{formattedMin}</td>}
+          {isMaxNumber && <td>{formattedMax}</td>}
         </tr>
       </tbody>
     </Table>
-  );
+  ) : null;
 }
 
 FieldFingerprintInfo.propTypes = propTypes;

--- a/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
+++ b/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.jsx
@@ -1,7 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
 import { t } from "ttag";
-import _ from "underscore";
 
 import { formatDateTimeWithUnit, formatNumber } from "metabase/lib/formatting";
 import Field from "metabase-lib/lib/metadata/Field";

--- a/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.unit.spec.js
+++ b/frontend/src/metabase/components/MetadataInfo/FieldFingerprintInfo/FieldFingerprintInfo.unit.spec.js
@@ -122,6 +122,46 @@ describe("FieldFingerprintInfo", () => {
         expect(screen.getByText("5")).toBeVisible();
       });
     });
+
+    describe("with empty type/Number fingerprint", () => {
+      beforeEach(() => {
+        numberField.fingerprint = {
+          type: {
+            "type/Number": {},
+          },
+        };
+
+        setup(numberField);
+      });
+
+      it("should render nothing", () => {
+        expect(container.firstChild).toBeNull();
+      });
+    });
+
+    describe("with missing type/Number property", () => {
+      beforeEach(() => {
+        numberField.fingerprint = {
+          type: {
+            "type/Number": {
+              min: 1,
+              max: 5,
+            },
+          },
+        };
+
+        setup(numberField);
+      });
+
+      it("should not render anything for the avg", () => {
+        expect(screen.queryByText("Average")).toBeNull();
+      });
+
+      it("should still render min and max", () => {
+        expect(screen.getByText("1")).toBeVisible();
+        expect(screen.getByText("5")).toBeVisible();
+      });
+    });
   });
 
   describe("Other field types", () => {

